src/group/membership/components/GroupMembershipCard.js

```diff
@@ -146,11 +146,11 @@ const GroupMembershipCard = props => {
   const { groupSlug } = useGroupContext();
   const { onViewMembers, InfoComponent } = props;
   const { fetching, group } = useSelector(
-    _.getOr({}, `group.memberships.${groupSlug}`)
+    state => state.memberships.memberships[groupSlug] ?? {}
```

Comment:
Suggestion: Should we change the state structure to: `state.memberships.list` or maybe even `state.memberships.dict` since it is a dictionary? `state.memberships.memberships` seems confusing to me

Reply:
i think that might be a good idea, maybe `state.memberships.lists` since it is a dict of `MembershipLists`

Reply:
That sounds good to me